InstallNotes/MakeDefsLocalExamples/Mac-Catalina-GCC.Make.defs.local

```diff
@@ -0,0 +1,34 @@
+# This make defs file is for Mac OS Catalina
+# It assumes that you have used homebrew to install
+# the relevant packages as indicated for the commands below
+# You should also have obtained up to date Xcode command line tools
+# using 'xcode-select --install'
+DIM             = 3
+DEBUG           = TRUE
+OPT             = TRUE
+PRECISION       = DOUBLE
+USE_64          = TRUE
+# brew install gcc
+CXX             = g++-9 -std=c++14
```

Comment:
Is the `-std=c++14` flag necessary? If the compiler is correctly detected by the Chombo makefile system, this flag should be automatically loaded from [Chombo/lib/mk/compiler/Make.defs.GNU](https://github.com/GRChombo/Chombo/blob/ad44560a2ebe9c8bc0c4dc7e85806710190ea780/lib/mk/compiler/Make.defs.GNU#L64).
```suggestion
CXX             = g++-9
```